src/s1b_opening_crawl.tw2

```diff
@@ -1,17 +1,17 @@
 ::Opening crawl
-$monospace[The year is 20XX. Wicked clown shit is the new law.
+$monospace[(live: 1s)[(if: time > 1s)[$dissolve[The year is 20XX. Wicked clown shit is the new law.(stop:)]]]
```

Comment:
Going straight from the end of the portapotty to this screen was jarring, since the first line was instantly visible. I think a small delay adds to the impact of that first line, and it really hits nicely while it stands alone for a couple seconds.